### PR TITLE
course-show list member

### PR DESCRIPTION
**Pull request by truonglocbinh**

### CODE REVIEW

File: app/assets/stylesheets/user_courses.css.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #course-show {
+  margin-top: -3%;
+
```
Comment (bapboy18) [inline app/assets/stylesheets/user_courses.css.scss]: remove this line

File: app/assets/stylesheets/user_courses.css.scss [hunk 2]
```diff
@@ -36,16 +38,60 @@
...
+    color: #aebdc1;
+  }
+
+  h3 {
+    text-align: center;
+    margin-top: 15px;
+    font-weight: 400;
+    font-size: 95%;
+  }
+}
```
Comment (bapboy18) [inline app/assets/stylesheets/user_courses.css.scss]: add space line end of file

File: app/views/user_courses/_list_member.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<div class="box box-primary">
+  <div class="box-header with-border">
+    <h3 class="box-title"><%= t "courses.tab.trainees" %></h3>
+  </div>
+  <div class="box-body no-padding">
+    <ul class="users-list clearfix">
+      <%= render partial: "user_course", collection: @trainees, as: :user %>
+    </ul>
+  </div>
+</div>
```
Comment (bapboy18) [inline app/views/user_courses/_list_member.html.erb]: add space line

File: app/views/user_courses/show.html.erb [hunk 4]
```diff
@@ -1,14 +1,18 @@
 <div class="row" id="course-show">
   <div class="col-md-9">
-    <div class="row course-breadcrumb">
+    <div class="row" id="course-breadcrumb">
       <ol class="breadcrumb">
         <li>
-          <%= link_to t("courses.breadcum.home"), root_url %>
+          <%= link_to root_url do %>
+            <i class="glyphicon glyphicon-th-list"></i>
+            <%= t("courses.paths") %>
```
Comment (thao33) [inline app/views/user_courses/show.html.erb]: remove ()

File: config/locales/course.ja.yml [hunk 5]
```diff
@@ -54,3 +54,5 @@ ja:
       progress: "進行中"
       finish: "完成"
     day: "日"
+    view_detail: "観点"
```
Comment (thao33) [inline config/locales/course.ja.yml]: 詳細

**Comment by vuducluan** [comment]
OK

**Comment by thao33** [comment]
Ok.

**Comment by vietmt** [comment]
OK

File: config/locales/course.ja.yml [hunk 6]
```diff
@@ -54,3 +54,5 @@ ja:
       progress: "進行中"
       finish: "完成"
     day: "日"
+    view_detail: "詳細"
```
Comment (thao33) [inline config/locales/course.ja.yml]: 詳細を見る

File: app/helpers/application_helper.rb [hunk 7]
```diff
@@ -72,9 +72,14 @@ def image_course_tag course, language
...
 
-  def image_course_subject_tag user_subject
+  def user_image_course_subject_tag user_subject
     image_tag user_subject.image_url ? user_subject.image_url : "no_image",
-      class: "img-responsive"
+      class: "img-circle img-responsive"
+  end
+
+  def image_user_course_avatar_tag user
+    image_tag user.avatar_url ? user.avatar_url : "profile",
```
Comment (dieunb) [inline app/helpers/application_helper.rb]: `if..else`

**Pull request merged**